volumina/widgets/dataExportOptionsDlg.py

```diff
@@ -415,7 +416,7 @@ def validate(self, userAxes, pos):
             userSet = set(str(userAxes))
 
             # Ensure all user axes appear in the input
-            if not (userSet <= inputSet):
+            if not inputSet.issubset(self._valid_axes):
```

Comment:
Shouldn't this be `if not userSet.issubset(self.__valid_axes)`?  Unless there's a weird bug elsewhere in ilastik, `inputSet` will always be a subset of `"tzyxc"`

Also, maybe fix the comment to something that reflects the change like "Ensure all axes are valid" =)

Reply:
ah yeah. I think naming here is suboptimal to invite input / user (user-input?!) confusion ;)